### PR TITLE
Add portal generated module init file

### PR DESCRIPTION
**Pull request by YalinLi0312**
related issue: https://github.com/Azure/azure-cli-extensions/issues/1608
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/portal/azext_portal/generated/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```
Comment (haroldrandom) [inline src/portal/azext_portal/generated/__init__.py:12]: I suggest not adding `__path__ = __import__('pkgutil').extend_path(__path__, __name__)` to chose one of Python Namesapce, leave it with License is ok.

Comment (YalinLi0312) [reply src/portal/azext_portal/generated/__init__.py:12]: I just use the same one as in logic. Will remove this line in next commit

**Comment by YalinLi0312** [comment]
> Could you upgrade the version in `setup.py` to `0.1.2` in order to trigger release pipeline to help release.

The current version of portal is `0.1.0`, the version should be `0.1.1` after upgrade, right?

**Comment by haroldrandom** [comment]
> > Could you upgrade the version in `setup.py` to `0.1.2` in order to trigger release pipeline to help release.
> 
> The current version of portal is `0.1.0`, the version should be `0.1.1` after upgrade, right?

Yeah, my mistake

**Pull request merged**